docs/guides/interfaces.rst

```diff
@@ -38,28 +34,32 @@ However, using abstract classes requires using inheritance, which sometimes is u
 
 Orquestra provides the following interfaces:
 
-* :class:`CircuitRunner <orquestra.quantum.api.circuit_runner.CircuitRunner>` (:ref:`guide <backends_guide>`)
-* :class:`WavefunctionSimulator <orquestra.quantum.api.wavefunction_simulator.WavefunctionSimulator>` (:ref:`guide <backends_guide>`)
-* :class:`Optimizer <orquestra.opt.api.Optimizer>` (:ref:`guide <optimizers_guide>`)
-* :class:`CostFunction <orquestra.opt.api.CostFunction>` (:ref:`guide <cost_function_guide>`)
-* :class:`Ansatz <orquestra.vqa.api.ansatz.Ansatz>` (:ref:`guide <ansatzes_guide>`)
-* :class:`Estimators <orquestra.quantum.api.estimation.EstimateExpectationValues>` (:ref:`guide <estimators_guide>`)
+
+.. list-table::
+    :align: left
+
+    * - :class:`CircuitRunner <orquestra.quantum.api.circuit_runner.CircuitRunner>` (:ref:`guide <backends_guide>`)
+    * - :class:`WavefunctionSimulator <orquestra.quantum.api.wavefunction_simulator.WavefunctionSimulator>` (:ref:`guide <backends_guide>`)
+    * - :class:`Optimizer <orquestra.opt.api.Optimizer>` (:ref:`guide <optimizers_guide>`)
+    * - :class:`CostFunction <orquestra.opt.api.CostFunction>`
+    * - :class:`Ansatz <orquestra.vqa.api.ansatz.Ansatz>` (:ref:`guide <ansatzes_guide>`)
+    * - :class:`Estimators <orquestra.quantum.api.estimation.EstimateExpectationValues>` (:ref:`guide <estimators_guide>`)
 
 
 Passing parameters
 ------------------
 
-You might wonder - what if I have two optimizers which take different sets of parameters? For example, you might want to use `L-BFGS-B <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html#optimize-minimize-lbfgsb>`_ and `COBYLA <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html#optimize-minimize-cobyla>`_ from `scipy`. This is how you would use them in scipy: 
+You might wonder - what if I have two optimizers which take different sets of parameters? For example, you might want to use `L-BFGS-B <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-lbfgsb.html#optimize-minimize-lbfgsb>`_ and `COBYLA <https://docs.scipy.org/doc/scipy/reference/optimize.minimize-cobyla.html#optimize-minimize-cobyla>`_ from ``scipy``. This is how you would use them in scipy: 
 
 .. literalinclude:: ../examples/guides/interfaces_guide.py
     :start-after: >> Guide code snippet: script showing scipy optimizers
     :lines: 2-3
     :language: python
 
 
-`L-BFGS-B` doesn't support argument called `constraints` and `COBYLA` doesn't support `bounds`. So how can you make them fit the same interface, if they have different inputs?
+``L-BFGS-B``` doesn't support argument called ``constraints`` and ``COBYLA`` doesn't support ``bounds``. So how can you make them fit the same interface, if they have different inputs?
```

Comment:
I think this is one too many "`"?

Reply:
Good catch, fixed in faec13157e2adf175e2306705f526a4ec977cca2